src/containers/edit-profile/password-security-tab/PasswordSecurityTab.tsx

```diff
@@ -1,227 +1,53 @@
-import { useCallback, FC } from 'react'
+import { FC } from 'react'
 import { useTranslation } from 'react-i18next'
-
 import Box from '@mui/material/Box'
-import Typography from '@mui/material/Typography'
-import Divider from '@mui/material/Divider'
-
-import Loader from '~/components/loader/Loader'
-import AppButton from '~/components/app-button/AppButton'
-import AppTextField from '~/components/app-text-field/AppTextField'
-import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
-import { snackbarVariants } from '~/constants'
 
-import useForm from '~/hooks/use-form'
+import { useModalContext } from '~/context/modal-context'
 import useChangeUserStatus from '~/hooks/use-change-user-status'
-import useAxios from '~/hooks/use-axios'
-import { useAppDispatch, useAppSelector } from '~/hooks/use-redux'
-import useInputVisibility from '~/hooks/use-input-visibility'
-import useConfirm from '~/hooks/use-confirm'
-
-import { AuthService } from '~/services/auth-service'
-import { openAlert } from '~/redux/features/snackbarSlice'
-
-import { styles } from '~/containers/edit-profile/password-security-tab/PasswordSecurityTab.styles'
+import PasswordSecurityItem from './password-security-item/PasswordSecurityItem'
```

Comment:
Can you organize the imports and use an alias `~`?

Reply:
organized